src/structures/GuildMember.js

```diff
@@ -302,6 +302,17 @@ class GuildMember extends Base {
     return this.manageable && (this.guild.me?.permissions.has(Permissions.FLAGS.MODERATE_MEMBERS) ?? false);
   }
 
+  /**
+   * Whether this member is currently timed out
+   * @returns {boolean}
+   */
+  isTimedOut() {
```

Comment:
Shouldnt this be named `isCommunicationDisabled`? There was a discussion about the naming in #7104

Reply:
thanks 👍